src/borg/testsuite/archiver/rlist_cmd.py

```diff
@@ -34,12 +35,12 @@ def test_archives_format(self):
         )
         self.assertEqual(output_1, output_2)
         output_1 = self.cmd(f"--repo={self.repository_location}", "rlist", "--short")
-        self.assertEqual(output_1, "test-1\ntest-2\n")
+        self.assertEqual(output_1, "test-1" + os.linesep + "test-2" + os.linesep)
         output_1 = self.cmd(f"--repo={self.repository_location}", "rlist", "--format", "{barchive}/")
         self.assertEqual(output_1, "test-1/test-2/")
         output_3 = self.cmd(f"--repo={self.repository_location}", "rlist", "--format", "{name} {comment}{NL}")
-        self.assert_in("test-1 comment 1\n", output_3)
-        self.assert_in("test-2 comment 2\n", output_3)
+        self.assert_in("test-1 comment 1" + os.linesep, output_3)
+        self.assert_in("test-2 comment 2" + os.linesep, output_3)
```

Comment:
instead of repeating `os.linesep` all over the place, I guess we could consider putting something like this into `borg.constants` from where we usually star-import everything:

```
LS = os.linesep
```

And then use `LS` (or `NL` like "newline") everywhere.

Reply:
can you do a global search whether that is the only place where `{NL}` is used?

Reply:
seen my comment?

for consistent, guess it should be `NL`.

Reply:
> can you do a global search whether that is the only place where `{NL}` is used?

I've sent some messages regarding this on IRC.

Reply:
answered there. also I'll have a look at the code right now myself...